integrations/aws/.port/resources/blueprints.json

```diff
@@ -0,0 +1,283 @@
+[
+  {
+    "identifier": "ec2Instance",
+    "description": "This blueprint represents an AWS EC2 instance in our software catalog",
+    "title": "EC2 Instance",
+    "icon": "EC2",
+    "schema": {
+      "properties": {
+        "architecture": {
+          "type": "string",
+          "title": "Architecture",
+          "enum": ["i386", "x86_64", "arm64", "x86_64_mac", "arm64_mac"]
+        },
```

Comment:
you can remove them from the defaults for now, but save them to add it as examples to the docs